static/swagger-specs/destinations.yaml

```diff
@@ -3313,10 +3313,21 @@ components:
         - interval
         - timeUnit
       properties:
+        exportMode:
+          title: Export mode
+          type: string
+          description: Select `DAILY_FULL_EXPORT` or `FIRST_FULL_THEN_INCREMENTAL`. `FIRST_FULL_THEN_INCREMENTAL` triggers an export of one or multiple files representing a full snapshot of the dataset. Subsequent files are incremental additions to the dataset since the previous export. `DAILY_FULL_EXPORT` can only be used in combination with `timeUnit`:`day` and `interval`:`0` for a one-time full export of the dataset. Daily full exports of datasets are not supported. If you need daily exports, use the incremental export option.
```

Comment:
One more tiny nit - we've got it as DAILY_FULL_EXPORT and FIRST_FULL_THEN_INCREMENTAL on the first sentence, but when we describe what they mean, we have it as FIRST_FULL_THEN_INCREMENTAL then DAILY_FULL_EXPORT. Let's make that consistent by having them both be in the same order.